### ISSUE TITLE
[Bug]: Nuxtr not working with new Nuxt 4 directory structure

### ISSUE THREAD

**Issue by Conobi** [nuxtrdev/nuxtr-vscode#150]
### Environment

------------------------------
- Operating System: Linux
- Node Version:     v22.2.0
- Nuxt Version:     3.12.3
- CLI Version:      3.12.0
- Nitro Version:    2.9.7
- Package Manager:  pnpm@9.4.0
- Builder:          -
- User Config:      devtools, future, modules, compatibilityDate
- Runtime Modules:  @nuxt/ui@2.17.0
- Build Modules:    -
------------------------------

### What went wrong?

Commands and useful tools from Nuxtr aren't working properly with the new Nuxt 4 directory structure.
See : https://nuxt.com/docs/getting-started/upgrade#new-directory-structure for details about these changes.

### How to reproduce it?

Enable:
```json
  future: {
    compatibilityVersion: 4
  },
```
In `nuxt.config.ts`

**Comment by BayBreezy** [comment]
I wouldn't say this is a bug... More of an enhancement that is needed

**Comment by gsxdsm** [comment]
Is there a plan for Nuxt 4 support?

**Comment by adhamfarrag** [comment]
@gsxdsm Yes, major release coming by the end of this month that includes Nuxt 4 support.

**Comment by gsxdsm** [comment]
Amazing! Looking forward to it

**Comment by alifnuryana** [comment]
Does this extension currently support the app directory?

**Comment by BabyLy233** [comment]
Is this extension still being maintained?

**Comment by adhamfarrag** [comment]
Hey @BabyLy233, yes. I'm currently working on a rewrite.

**Comment by Deeyuude** [comment]
Hello There 
Any updates please ?

**Comment by zuhailkhan** [comment]
Is there an update on this issue yet?
